kornia/geometry/conversions.py

```diff
@@ -340,16 +354,22 @@ def rotation_matrix_to_quaternion(
 
     Example:
         >>> input = torch.rand(4, 3, 3)  # Nx3x3
-        >>> output = rotation_matrix_to_quaternion(input)  # Nx4
+        >>> output = rotation_matrix_to_quaternion(input, eps=torch.finfo(input.dtype).eps, order=QuaternionCoeffOrder.WXYZ)  # Nx4
     """
     if not isinstance(rotation_matrix, torch.Tensor):
-        raise TypeError("Input type is not a torch.Tensor. Got {}".format(
-            type(rotation_matrix)))
+        raise TypeError(f"Input type is not a torch.Tensor. Got {type(rotation_matrix)}")
 
     if not rotation_matrix.shape[-2:] == (3, 3):
-        raise ValueError(
-            "Input size must be a (*, 3, 3) tensor. Got {}".format(
-                rotation_matrix.shape))
+        raise ValueError(f"Input size must be a (*, 3, 3) tensor. Got {rotation_matrix.shape}")
+
+    if order not in QuaternionCoeffOrder.ALL:
```

Comment:
@amonszpart @shijianjian apparently jit doesn't like this comparison :)

Reply:
So I can change from .ALL to != WXYZ, and ~= XYZW, but it still complains because of the str value of the non Enum. 
I have very little experience with JIT, so not really the person to fix this...
I tried `global QuaternionCoeffOrder`, but JIT doesn't like globals. 

```shell
E           RuntimeError: 
E           python value of type 'str' cannot be used as a value. Perhaps it is a closed over global variable? If so, please consider passing it in as an argument or use a local varible instead.:
E             File "/home/aron/workspace/kornia/kornia/geometry/conversions.py", line 639
E                   raise ValueError(f"Input must be a tensor of shape (*, 3). Got {quaternion.shape}")
E           
E               if order != QuaternionCoeffOrder.XYZW and order != QuaternionCoeffOrder.WXYZ:
E                           ~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
E                   raise ValueError(f"order must be one of {QuaternionCoeffOrder.ALL}")

../../.dev_env/envs/venv/lib/python3.7/site-packages/torch/jit/_script.py:990: RuntimeError
```